### PR TITLE
[FIX] web: searchPanel style tweaks for the panel selectors

### PR DESCRIPTION
**Pull request by ThanhDodeurOdoo**
Moves the custom documents search panel style to the search panel
of community to improve usability and fix several visual glitches.

Task: #2061957

### CODE REVIEW

File: addons/web/static/src/scss/search_panel.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 // ------- View with SearchPanel -------
+$o-searchpanel-active-bg: rgba(108, 193, 237, 0.5);
+$o-searchpanel-p-custom: 4px;
```
Comment (alexkuhn) [inline addons/web/static/src/scss/search_panel.scss]: I wouldn't make a variable for that

File: addons/web/static/src/scss/search_panel.scss [hunk 2]
```diff
@@ -31,6 +33,7 @@ $o-searchpanel-filter-default-color: #D59244;
         }
 
         .o_search_panel_label {
+            padding-right: $o-searchpanel-p-custom +4;
```
Comment (alexkuhn) [inline addons/web/static/src/scss/search_panel.scss]: missing `px`

File: addons/web/static/src/scss/search_panel.scss [hunk 3]
```diff
@@ -56,13 +59,18 @@ $o-searchpanel-filter-default-color: #D59244;
...
         .o_search_panel_category_value {
             header {
-                margin-left: -$o-searchpanel-p-tiny;
-                padding-left: $o-searchpanel-p-tiny;
+                margin-top: -$o-searchpanel-p-custom + 1;
+                margin-bottom: -$o-searchpanel-p-custom + 1;
+                padding: $o-searchpanel-p-custom $o-searchpanel-p-custom +3;
+
+                .o_toggle_fold {
+                    padding-left: 6%;
```
Comment (alexkuhn) [inline addons/web/static/src/scss/search_panel.scss]: IMO would be better with `px`, instead of `%`

File: addons/web/static/src/xml/kanban.xml [hunk 4]
```diff
@@ -243,9 +243,9 @@
         <t t-set="value" t-value="category.values[valueId]"/>
         <li class="o_search_panel_category_value border-0 list-group-item" t-att-data-id="value.id" t-att-data-category-id="category.id">
             <header t-att-class="'list-group-item-action' + (value.id === category.activeValueId ? ' active' : '')">
-                <label t-att-for="value.display_name" class="o_search_panel_label mb0 d-block">
-                    <i t-if="value.childrenIds.length" t-att-class="'fa fa-fw pull-right o_toggle_fold ' + (value.folded ? 'fa-caret-left' : 'fa-caret-down')"/>
+                <label t-att-for="value.display_name" class="o_search_panel_label mb0">
                     <span class="o_search_panel_label_title"><t t-esc="value.display_name"/></span>
+                    <i t-if="value.childrenIds.length" t-att-class="'fa fa-fw o_toggle_fold ' + (value.folded ? 'fa-caret-left' : 'fa-caret-down')"/>
```
Comment (alexkuhn) [inline addons/web/static/src/xml/kanban.xml]: `t-attf`

**Comment by ThanhDodeurOdoo** [comment]
robodoo retry

**Pull request closed without merging**